### PR TITLE
My hw 01

### PR DESCRIPTION
**Pull request by TanjaZzz**

### CODE REVIEW

File: HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_01.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+puts 'Welcome to my first program!'
+print 'Type hours'
+h = gets.chomp.to_i
+print 'Type minutes'
+m = gets.chomp.to_i
+puts 'Awesome! Let me work hard instead of you!'
+puts "The angle is #{(30 * h - 5.5 * m).abs} degrees"
```
Comment (vladhilko) [inline HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_01.rb]: very clean 👍

File: HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+6 / (1 - 3.00 / 4) == 24
```
Comment (vladhilko) [inline HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb]: `you can only divide, multiply or add them` not sure that we can use subtraction here

Comment (TanjaZzz) [reply HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb]: Should I replace then the code to 6 * 4 + 1 / 3 == 24 ?

File: HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_01.rb (hunk 1 above)
Comment (TanjaZzz) [reply HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_01.rb]: cool! thanks! :)

File: HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb (hunk 2 above)
Comment (vladhilko) [reply HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb]: I think, yes, it would be more correct

Comment (TanjaZzz) [reply HW01/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_01_t_02.rb]: done!

**Pull request merged**